types/promise-fs/index.d.ts

```diff
@@ -0,0 +1,333 @@
+// Type definitions for promise-fs 2.1
+// Project: https://github.com/octet-stream/promise-fs#readme
+// Definitions by: Daniel Cassidy <https://github.com/djcsdy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { PathLike, WriteFileOptions, RmDirAsyncOptions, MakeDirectoryOptions, Dirent, Stats } from "fs";
+
+export {
+    PathLike, StatsBase, Stats, Dirent, FSWatcher, ReadStream, WriteStream, renameSync, truncateSync,
+    ftruncateSync, chownSync, fchownSync, lchownSync, chmodSync, fchmodSync, lchmodSync, statSync,
+    fstatSync, lstatSync, linkSync, symlinkSync, readlinkSync, realpathSync, unlinkSync, RmDirOptions,
+    RmDirAsyncOptions, rmdirSync, MakeDirectoryOptions, mkdirSync, mkdtempSync, readdirSync, closeSync,
+    openSync, utimesSync, futimesSync, fsyncSync, writeSync, readSync, readFileSync, WriteFileOptions,
+    writeFileSync, appendFileSync, watchFile, unwatchFile, watch, existsSync, constants, accessSync,
+    createReadStream, createWriteStream, fdatasyncSync, copyFileSync, writevSync
+} from "fs";
```

Comment:
This should just be:
```suggestion
export * from "fs";
```

Reply:
No it shouldn’t. promise-fs doesn’t re-export the asynchronous functions in fs.

Reply:
Except that `export * from` doesn’t export what is overridden in the current module, and it correctly describes what `promise‑fs` does, which your code doesn’t.